dnaMap.js

```diff
@@ -1,72 +1,93 @@
+  }
+} // adds convertString func to enter keyboard key 
+
+function convert(inputStr, strandType) {
+  let bases = strandType == "DNA" ? ["C", "G", "A", "T"] : ["C", "G", "A", "U"];
+  const dnaMap = generateMap(bases);
 
-function convert(inputStr, strandType) {    
-   let bases = strandType == "DNA" ? ['C', 'G', 'A', 'T'] : ['C', 'G', 'A', 'U'];
-   const dnaMap = generateMap(bases);
-   
-   const dnaString = inputStr.split("").map(
-     (letter, i) => dnaMap[inputStr.charCodeAt(i)-32]).join("");
-   
-     return dnaString;
+  const dnaString = inputStr
+    .split("")
+    .map((letter, i) => dnaMap[inputStr.charCodeAt(i) - 32])
+    .join("");
+
+  return dnaString;
 }
 
-   function complement(inputStrand) {
-       const comps = {"a":"t", "t":"a", "c":"g", "g":"c",
-                       "A":"T", "T":"A", "C":"G", "G":"C"};
+function complement(inputStrand) {
+  const comps = {
+    a: "t",
+    t: "a",
+    c: "g",
+    g: "c",
+    A: "T",
+    T: "A",
+    C: "G",
+    G: "C"
+  };
 
-       let complement = "";
+  let complement = "";
 
-       for (i = 0; i < inputStrand.length; i++) {
-           if (comps[inputStrand[i]]) {
-               complement += comps[inputStrand[i]];
-           } else {
-               complement = "Invalid input strand";
-               break;
-           }
-       }
+  for (i = 0; i < inputStrand.length; i++) {
+    if (comps[inputStrand[i]]) {
+      complement += comps[inputStrand[i]];
+    } else {
+      complement = "Invalid input strand";
+      break;
+    }
+  }
 
-       return complement;
-   }
+  return complement;
+}
 
 function generateMap(bases) {
-   let dnaMap = [];
+  let dnaMap = [];
+
+  for (let i = 32, c = [0, 0, 0, 0, 0]; i <= 127; i++) {
+    let seq = "";
+    seq += bases[c[0]];
+    seq += bases[c[1]];
+    seq += bases[c[2]];
+    seq += bases[c[3]];
+    seq += bases[c[4]];
+
+    dnaMap.push(seq);
 
-   for (let i = 32, c = [0, 0, 0, 0, 0]; i <= 127; i++) {
-       let seq="";
-       seq += bases[c[0]];
-     seq += bases[c[1]];
-     seq += bases[c[2]];
-     seq += bases[c[3]];
-     seq += bases[c[4]];
-     
-     dnaMap.push(seq);
-     
-       if (c[4] == 3) {
-         c[4] = 0;
-       if (c[3] == 3) {
-         c[3] = 0;
-         if (c[2] == 3) {
-           c[2] = 0;
-           if (c[1] == 3) {
-             c[1] = 0;
-             if (c[0] == 3)	c[0] = 0;
-             else c[0]++;
-   
-           } else c[1]++;
-         } else c[2]++;
-       } else c[3]++;
-     } else c[4]++;
-   }
+    if (c[4] == 3) {
+      c[4] = 0;
+      if (c[3] == 3) {
+        c[3] = 0;
+        if (c[2] == 3) {
+          c[2] = 0;
+          if (c[1] == 3) {
+            c[1] = 0;
+            if (c[0] == 3) c[0] = 0;
+            else c[0]++;
+          } else c[1]++;
+        } else c[2]++;
+      } else c[3]++;
+    } else c[4]++;
+  }
 
-   return dnaMap;
+  return dnaMap;
 }
 
-module.exports = { convert };
\ No newline at end of file
+// module.exports = { convert };
```

Comment:
@rurru is this necessary?

Reply:
Yes, this is what connect the file to the test 😄

Reply:
was added back in. if thats all merge plz